training/trillium/Llama3-70B-PyTorch/XPK/profile_convert.py

```diff
@@ -0,0 +1,53 @@
+import xplane_pb2
```

Comment:
Can we put this script in a shared location (e.g. `utils/profile_convert.py` at the root of the repo)? I suspect it will be useful for many other models.

Reply:
SG!

Reply:
Moved to `tpu-recipes/utils/`. Instructions updated to call the script.